### ISSUE TITLE
BUGs in v0.1

### ISSUE THREAD

**Issue by leloup314** [cyclotron-bonn/nano_util_board#16]
- Footprints of resistors appear to be 0201 instead of 0603 
- Voltage divider potentials are switched: we used to measured the voltage drop over the NTC, now we measure over the fixed resistor (easily fixed in firmware)
- Maybe I2C issue (not working so far)

We received 4 PCBs, the NTC readout is wroking fine, the I2C to the R/O board is not yet working, the frequency counting needs some testing ;)

**Comment by leloup314** [comment]
Also: when ordering the PCB, the company said that the _drill information_ was missing in the Gerber files (which were then added by our electronicslab)

**Comment by BelaRincewind** [comment]
Alright, I'll work on it. Are there any other enhancements I should add? See issue #15.

**Comment by BelaRincewind** [comment]
> Also: when ordering the PCB, the company said that the _drill information_ was missing in the Gerber files (which were then added by our electronicslab)

Oh, sorry for that. There is indeed an option for that in the export menu. So that should be a none issue next time

**Comment by BelaRincewind** [comment]
Should I swap NTC and 10k so that it'll be as intended before?

**Comment by leloup314** [comment]
Yes please
  

  

  

  
  
  
  
  
>

**Comment by BelaRincewind** [comment]
Hopefully I fixed those issues. For now, I did **not** change the connection to the I2C RJ45, as I don't know how the correct pinout is. But changing this is no biggy as just the lanes would need to be changed.

**Comment by leloup314** [comment]
I have the feeling that the I2C bus on the PCB is not working properly. We have a BA student who exactly uses NTC readout and I2C at the same time, using a breadboard. There it works whereas it does not with the PCB. So maybe the pinout for the RJ45 is correct but I2C is generally not working. Would be good if you meet with the BA student to check the breadboard schematic

**Comment by leloup314** [comment]
@bedakno can you comment on the status of the PCB? Maybe a summary of what bugs have been identified and fixed and what has been tested?

**Comment by BelaRincewind** [comment]
Current status: I2C (overlapping of SDA and Vin) has been fixed in v0.2, 10k pullup-resistors are included. Also all footprint sizes are now 0603, the voltage divider has been swapped, I2C Vcc select was erased, in favour of selection pins for different use cases. Currently not tested is the frequency counting. This is will be done this week. I have also drafted a release for v0.2 where I will be continuously adding fixes and their original bugs as well as other enhances or changes.
